src/main/kotlin/org/rust/cargo/util/RustCrateUtil.kt

```diff
@@ -30,7 +30,7 @@ enum class StdLibType {
 data class StdLibInfo (
     val name: String,
     val type: StdLibType,
-    val srcDir: String = "lib$name",
+    val possibleSrcDirs: List<String> = listOf("lib$name", "$name/src"),
```

Comment:
Can we modify this code somehow to provide `"lib$name"` or `name` as a package dir instead of `src` folder?
It's more consistent with libraries from cargo. Also, it allows us to provide more user friendly structure of stdlib library

| stable | nightly |
| - | - |
| <img width="376" alt="Screen Shot 2020-08-02 at 14 39 18" src="https://user-images.githubusercontent.com/2539310/89122256-818bdd80-d4ce-11ea-994c-6525a095d6a6.png"> | <img width="375" alt="Screen Shot 2020-08-02 at 14 40 04" src="https://user-images.githubusercontent.com/2539310/89122260-881a5500-d4ce-11ea-8c64-f763345ea5d5.png"> |

Reply:
Fixed.